CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
 
 ## master (unreleased)
 
+### Bug fixes
```

Comment:
You shouldn't edit the changelog manually - it's generated from the files in the `changelog` folder at release time.

Reply:
Sorry, my mistake. Is this the correct way?